foreman/dynflow-executor.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman jobs daemon
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+User=foreman
+TimeoutSec=600
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/dynflow-executor start
+ExecStop=/usr/bin/dynflow-executor stop
+EnvironmentFile=-/etc/sysconfig/dynflow-executor
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
What about `Alias=foreman-tasks`, at least until we have the other tooling (such as katello-service) updated to the new name?

Reply:
Sounds good to me, will do

Reply:
(done)